tenants/all/templates/am-lfte.marko

```diff
@@ -97,7 +97,7 @@ $ const resolvedToNodesConverter = ({ resolved }) => (resolved.map((node) => ({
           with-image=false
           with-section=true
           skip=3
-          limit=6
+          limit=14
```

Comment:
This makes the number of articles pulled for "Main" 17, is this intended? 1+1+1+14 =17

Reply:
@Shinsina Nope, wrong math!